nixos/modules/services/admin/nomachine.nix

```
@@ -0,0 +1,247 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  cfg = config.services.nxserver;
+  nxserver = pkgs.nomachine; 
+
+  boolToStr = value: if value then "0" else "1";
+
+  serverConf = ''
+        ConfigFileVersion 4.0
+
+        EnableDebug ${boolToStr cfg.enableDebug}
```

```suggestion
        EnableDebug ${toString cfg.enableDebug}
```

`toString` converts `true` to `"1"`.